data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/beacon/PostBlsToExecutionChanges.java

```diff
@@ -76,6 +76,13 @@ private static EndpointMetadata createEndpointMetadata(final SchemaDefinitionCac
   @Override
   public void handleRequest(RestApiRequest request) throws JsonProcessingException {
     final List<SignedBlsToExecutionChange> blsToExecutionChanges = request.getRequestBody();
+    if (blsToExecutionChanges.size() > 5000) {
```

Comment:
Can we move this into a constant? Something like `MAX_BLS_MESSAGER_PER_REQUEST` or similar?